### DISCUSSION TITLE
classes/class_charfxtransform

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#210] [User-contributed notes]
# classes/class_charfxtransform

Inherits: RefCounted< Object Controls how an individual character will be displayed in a RichTextEffect. Description: By setting various properties on this object, you can control how individual ch...

https://docs.godotengine.org/en/stable/classes/class_charfxtransform.html

**Comment by sky94520** [comment]
According to the github comments: https://github.com/godotengine/godot/issues/71095#issuecomment-1378651952, the param: visible doesn't reflow the characters.

**Comment by Leroymilo** [comment]
Running 4.4.stable, the `env` attribute does not convert arguments properly: for example, the custom bbcode tag `[tool count=2]` will convert the count value to a float instead of an int, which is not what this documentation suggests. I don't know if this is worth opening a bug report.

**Reply by Calinou** [reply]
Please open an issue on https://github.com/godotengine/godot/issues with a minimal reproduction project attached.

**Reply by Leroymilo** [reply]
Done: https://github.com/godotengine/godot/issues/104506
